plugins/modules/subscription_info.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021 Evgeni Golov
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: subscription_info
+version_added: 2.1.0
+short_description: Fetch information about Subscriptions
+description:
+  - Fetch information about Subscriptions
+author:
+  - "Evgeni Golov (@evgeni)"
+extends_documentation_fragment:
+  - theforeman.foreman.foreman
+  - theforeman.foreman.foreman.infomodule
+'''
+
+EXAMPLES = '''
+- name: "Show a subscription"
+  theforeman.foreman.subscription_info:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    name: "Red Hat Satellite Infrastructure Subscription"
```

Comment:
No organization context is needed to fetch sub info?

Reply:
Can we have one or all of the tests reflect that so we are testing the example? (e.g. drop https://github.com/theforeman/foreman-ansible-modules/pull/1223/files#diff-26c401d211772be08e88a4e26eb7f42fc1c21f16b49ace60767e3113a237b49eR29)

Reply:
dropped from all tests